src/main/java/UseCaseClasses/ScheduleEditor.java

```diff
@@ -21,18 +24,14 @@ public void addEvent(String eventType, String eventName, String eventDay, int ev
             event.setEventStartTime(eventStartTime);
             event.setEventEndTime(eventEndTime);
 
-            HashMap<Integer, String> Day = schedule.schedule.get(eventDay);
+            HashMap<String, String> Day = schedule.schedule.get(eventDay);

```

Comment:
if we need access to the Schedule class's instance attribute 'schedule', we have to make a getter method (relates to earlier comment about absolutely not letting an instance variable be public)